File: 프로그래머스/unrated/181830. 정사각형으로 만들기/정사각형으로 만들기.java
class Solution {
    public int[][] solution(int[][] arr) {
        int a = arr.length;
        int b = arr[0].length;
        
        if (a == b) {
            return arr;
        }
        
        int c = Math.max(a,b);
        
        int[][] ans = new int[c][c];
        
        for (int i = 0; i < arr.length; i++) {
            for (int j = 0; j < arr[0].length; j++) {
                ans[i][j] = arr[i][j];
            }
        }

        return ans;
    }
}

File: 프로그래머스/unrated/181830. 정사각형으로 만들기/정사각형으로 만들기.py
def solution(arr):
    a = len(arr)
    b = len(arr[0])
    
    if a == b:
        return arr
    
    c = max(a,b)
    ans = [[0]*c for _ in range(c)]
            
    for i in range(a):
        for j in range(b):
            ans[i][j] = arr[i][j]
    
    return ans